pkg/sdk/testint/users_integration_test.go

```diff
@@ -79,6 +79,39 @@ func TestInt_UsersShow(t *testing.T) {
 	})
 }
 
+func TestInt_UserAlter(t *testing.T) {
+	client := testClient(t)
+	ctx := testContext(t)
+
+	randomPrefix := random.AlphaN(6)
```

Comment:
nit: the prefix is not needed
And It would be the best to validate the policy is attached to the user after successful alter, but I think we don't support it right now in the SDK, so let's leave it as is.
cc: @sfc-gh-asawicki

Reply:
yes, please add TODO with SNOW-1645875

Reply:
Addressed in https://github.com/Snowflake-Labs/terraform-provider-snowflake/pull/3068